service/resource/deploymentv3/update.go

```diff
@@ -75,22 +74,10 @@ func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desir
 	}
 
 	var deploymentsToUpdate []*v1beta1.Deployment
+
 	if updateallowedcontext.IsUpdateAllowed(ctx) {
 		r.logger.LogCtx(ctx, "debug", "finding out which deployments have to be updated")
 
-		// Check if config maps of deployments changed. In case they did, add the
-		// deployments to the list of deployments intended to be updated.
-		m, ok := messagecontext.FromContext(ctx)
-		if ok {
-			for _, name := range m.ConfigMapNames {
-				desiredDeployment, err := getDeploymentByConfigMapName(desiredDeployments, name)
-				if err != nil {
-					return nil, microerror.Mask(err)
-				}
-				deploymentsToUpdate = append(deploymentsToUpdate, desiredDeployment)
-			}
-		}
```

Comment:
We are not acting upon configmap changes within the deployment resources anymore. The responsibility to trigger updates will be given to the version bundle version a guest cluster is configured for. The usual reconciliation of configmaps and deployments is still functional, but the trigger for an update is made more straight towards the use of version bundles.